deployment/v2/4_createRollup.ts

```diff
@@ -69,16 +70,23 @@ async function main() {
         sovereignParams,
     } = createRollupParameters;
 
-    const supportedConsensus = ["PolygonZkEVMEtrog", "PolygonValidiumEtrog", "PolygonPessimisticConsensus"];
+    const arraySupportedAggchains = ["AggchainECDSA", "AggchainFEP"];
+    const arraySupportedConsensus = ["PolygonZkEVMEtrog", "PolygonValidiumEtrog", "PolygonPessimisticConsensus"];
```

Comment:
```suggestion
    const arraySupportedLegacyConsensus = ["PolygonZkEVMEtrog", "PolygonValidiumEtrog", "PolygonPessimisticConsensus"];
```

Just to explicitly differentiate between OLD (StateTransistionChains & PPv0.2.0) & v0.3.0.
I think worth also adding the comment above xD